asset/quickstart/update-saved-query/main.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+ 
+// [START asset_quickstart_update_saved_query]
+ 
+// Sample update-saved_query update saved query.
+package main
+ 
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+ 
+	asset "cloud.google.com/go/asset/apiv1"
+	"cloud.google.com/go/asset/apiv1/assetpb"
+	cloudresourcemanager "google.golang.org/api/cloudresourcemanager/v1"
+	field_mask "google.golang.org/genproto/protobuf/field_mask"
+)
+ 
+// Command-line flags.
+var (
```

Comment:
Do not use command-line set up or flags:
https://googlecloudplatform.github.io/samples-style-guide/#no-cli

Reply:
changed.